packages/patrol_cli/lib/src/runner/patrol_command_runner.dart

```diff
@@ -380,24 +397,16 @@ Ask questions, get support at https://github.com/leancodepl/patrol/discussions''
   }
 
   void _handleAnalytics() {
-    _logger.info(
-      '''
-\n
-+---------------------------------------------------+
-|             Patrol - Ready for action!            |
-+---------------------------------------------------+
-| We would like to collect anonymous usage data     |
-| to improve Patrol CLI. No sensitive or private    |
-| information will ever leave your machine.         |
-+---------------------------------------------------+
-\n''',
-    );
-    final analyticsEnabled = _logger.confirm(
-      'Enable analytics?',
-      defaultValue: true,
-    );
-    _analytics.enabled = analyticsEnabled;
-    if (analyticsEnabled) {
+    _logger.info(_helloPatrol);
+
+    /// If the environment variable `PATROL_ANALYTICS_ENABLED` is set,
+    /// use it to determine if the command should be sent.
+    /// If not, analytics will be enabled by default.
+    final patrolAnalyticsEnabled =
+        p.Platform.environment['PATROL_ANALYTICS_ENABLED'];
```

Comment:
I would put `PATROL_ANALYTICS_ENABLED` into a const variable to prevent misspelling.